PackageViewModel/SymbolValidation/SymbolValidator.cs

```diff
@@ -258,51 +258,66 @@ private async Task CalculateValidity(IReadOnlyList<PackageFile> files)
 
             var requireExternal = false;
             // See if any pdb's are missing and check for a snupkg on NuGet.org. 
-            if (noSymbols.Count > 0 && _publishedOnNuGetOrg)
+            if (noSymbols.Count > 0)
             {
-                // try to get on NuGet.org
-                // https://www.nuget.org/api/v2/symbolpackage/Newtonsoft.Json/12.0.3 -- Will redirect               
-
-                
                 try
                 {
+                    // try to find a sibling snupkg file locally
+                    var directoryName = Path.GetDirectoryName(_packageViewModel.PackagePath) ?? string.Empty;
+                    var fileNameWithoutExtension = Path.GetFileNameWithoutExtension(_packageViewModel.PackagePath);
+                    var snupkgFilePath = Path.Combine(directoryName, fileNameWithoutExtension + ".snupkg");
```

Comment:
Nit: there’s a method Path.ChangeFileExtension that does these few lines

Reply:
Much nicer, thanks!